src/test/ui/asm/asm-misplaced-option.rs

```diff
@@ -21,14 +21,14 @@
 // ignore-mips
 // ignore-mips64
 
-#![feature(asm, rustc_attrs)]
-
+// compile-pass
+// skip-codegen
```

Comment:
Wait, can this be tested without using any part of LLVM? That's what `skip-codegen` do.

Reply:
We still get the warnings encoded in asm-misplaced-option.stderr while doing skip-codegen 

Is there something else you expect to be tested here?

Reply:
I didn't expect rustc to have an asm parser (and I thought that was done by LLVM), but I guess I'm wrong.

Reply:
... I think this test is *just* testing an aspect of how the macro arguments are being fed in.
I don't think there's any testing of the actual contents of the asm block.

Note in particular that this test was previously using `#[rustc_error]`, which means we've *never* been feeding it into LLVM; its always been erroring before code generation would start.

Reply:
Ah OK, that makes sense.